test/Integration/SqlInputBindingIntegrationTests.cs

```diff
@@ -98,6 +98,25 @@ public async void GetProductsNameEmptyTest(int n, int cost)
             Assert.Equal(expectedResponse, actualResponse, StringComparer.OrdinalIgnoreCase);
         }
 
+        [Fact]
+        public async void GetProductsByCostTest()
+        {
+            this.StartFunctionHost(nameof(GetProductsStoredProcedureFromAppSetting));
+
+            // Generate T-SQL to insert n rows of data with cost
+            Product[] products = GetProductsWithSameCost(1, 100);
```

Comment:
You should also insert some products with cost != 100 to make sure we're only grabbing the expected products